scrapper.py

```diff
@@ -29,6 +32,13 @@ def __init__(self, seed_urls, max_articles: int):
         pass
 
     def _extract_url(self, article_bs):
+        response = requests.get('https://lingngu.elpub.ru/jour')
+        with open('index.HTML', 'w', encoding='utf-8') as file:
+            file.write(response.text)
+        soup = BeautifulSoup(response.text, 'html.parser')
+        all_links = soup.find_all('a', attrs={'href': re.compile("^http[s]?://")})
```

Comment:
all links should be found in find_articles method. the extract_url method only extracts one url at a time